src/slashcommands/DefaultVc.ts

```diff
@@ -78,6 +78,7 @@ export class DefaultVc implements SlashCommand {
 export async function launchVoice(bot: Bot): Promise<void> {
 	defaultVc.forEach((channel, guild) => {
 		let guildCheck = bot.client.guilds.cache.get(guild.toString()) as Guild;
+		if (!guildCheck) return; //TODO: maybe delete the guild from the enmap?
```

Comment:
Yeah have it delete the guild from the enmap. Sound safer than doing the check every time for guilds it's not in